src/bin/genemichaels.rs

```diff
@@ -205,7 +240,47 @@ fn main() {
             Offable::On(_) => true,
         },
     };
-    if args.files.is_empty() {
+    if args.project {
+        let res = || -> Result<()> {
+            {
+                let inst = time::Instant::now();
+                eprintln!("\x1B[1;32m  Formatting\x1B[0;22m workspace...");
+                let mut project_cargo_toml = None;
+                let c_dir = current_dir()?;
+                let mut at: Option<&Path> = Some(&c_dir);
+                while let Some(d) = at.take() {
+                    if d.join(CARGO_TOML).exists() &&
```

Comment:
Would you mind putting these in a variable to avoid the repetition?

Reply:
Done.